tensorflow/core/common_runtime/graph_constructor.cc

```diff
@@ -96,6 +96,9 @@ class GraphConstructor {
         : allow_internal_ops(in.allow_internal_ops),
           expect_device_spec(in.expect_device_spec),
           importing(false),
+          uniquify_names(false),
+          uniquify_prefix(false),
+          skip_mapped_nodes(false),
```

Comment:
Sorry to come back to this but the merge blocked because of our internal compiler being a bit picky:

```
tensorflow/core/common_runtime/graph_constructor.cc:98:11: error: field 'importing' will be initialized after field 'uniquify_names' [-Werror,-Wreorder-ctor]
```

Could you re-order these?

Reply:
Done.